charts/backstage/values.yaml

```diff
@@ -121,6 +121,15 @@ backstage:
   extraVolumeMounts: []
   extraVolumes: []
 
+  # Allows to define the appConfig as a multiline string that generates a ConfigMap
+  # automatically, not requiring to have it pre provisioned as with the extraAppConfig key.
+  # DO NOT USE if you need to put sensitive data in the appConfig.
```

Comment:
Can we solve this by storing the `app-config.yaml` as a `Secret`?

Reply:
Well, the point is about having the configuration directly on your git repository (where I expect people will store their own values.yaml file), not that much about the ConfigMap. But yes, in case sensitive data can come here, it could be stored in a secret instead, good point :)

Reply:
yeah switching to secret won't help in this case. BTW we should mention this new feature in https://github.com/backstage/charts#configure-your-backstage-instance

Reply:
Just pushed it in https://github.com/backstage/charts/pull/23/commits/1ac59e91c59009e2a47e8766069a38ee6d43b782 @vinzscam  ;)